docs/writing_plugins/api.rst

```diff
@@ -712,6 +664,62 @@ Example:
     var $buttons = wym.get$Buttons();
 
 *******
+Dialogs
+*******
+
+``dialog(dialogObject)``
```

Comment:
@winhamwr I've changed the API so that the official way is to expect an object. This seems the most straightforward way to me.